src/calculateRentalCost.js

```diff
@@ -28,7 +28,17 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCost = days * 40;
```

Comment:
1. create variables to all numeric values to avoid `magic` numbers
https://www.inf.unibz.it/~calvanese/teaching/04-05-ip/lecture-notes/uni04/node15.html

2. use `const` for a base rent
```
const baseConst = days * baseRent
```

Reply:
fixed